xbmc/guilib/Texture.cpp

```diff
@@ -441,3 +442,13 @@ bool CBaseTexture::HasAlpha() const
 {
   return m_hasAlpha;
 }
+
+void CBaseTexture::SetMipmapping(bool mipmapping)
```

Comment:
is there a valid scenario which requires setting this to false? if so, how do you do this after the texture has been loaded? in case you don't have answers, I would just drop the parameter